homeassistant/components/vallox/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Vallox",
+        "description": "Set up the Vallox integration. If you have problems with configuration go to https://www.home-assistant.io/integrations/vallox.",
```

Comment:
Links are best to put in a description placeholder to avoid translation of the link.